packages/ember-flight-icons/package.json

```diff
@@ -23,6 +23,8 @@
   "scripts": {
     "build": "ember build --environment=production",
     "lint": "npm-run-all --aggregate-output --continue-on-error --parallel \"lint:!(fix)\"",
+    "lint:css": "stylelint \"**/*.css\"",
+    "lint:css:fix": "concurrently \"npm:lint:css -- --fix\"",
```

Comment:
for posterity: this was failing because I accidentally introduced CSS linting for the `ember-flight-icons` package – thanks @Dhaulagiri for spotting this!